docs/manuals/gui/viselements/file_download.md_template

```diff
@@ -37,11 +41,13 @@ Allows downloading _content_ when content is a file path or some content.
 
 ### Standard configuration
 
-A specific _label_ can be shown beside the standard icon.
+The [*label*](#p-label) property can be set to specify the label used as the button text next to
+the download icon.
 
-The function name provided as _on_action_ is called when the user initiates the download.
+The function name provided as *on_action* is called when the user initiates the download.
```

Comment:
not initiates except if content is None ?

Reply:
Correct!